shuttle/tests/it/shuttle_api.rs

```diff
@@ -359,7 +359,7 @@ async fn test_timestamp_interval() {
 }
 
 // <https://github.com/foxar-rs/foxar/issues/2341>
-#[ignore = "Forking is disabled"]
+#[ignore = "We do not support BSC"]
```

Comment:
I think we can remove this entire test, no reason to keep it around since it is for BSC.

Reply:
Sure